src/main/resources/bfa-config.properties

```diff
@@ -0,0 +1,2 @@
+webdriver.gecko.driver=/Users/ninhdoan/Desktop/selenium_drivers/firefox/geckodriver
+webdriver.chrome.driver=/Users/ninhdoan/Desktop/selenium_drivers/chrome/chromedriver
```

Comment:
Done. It is allowed to override the .proeprties configuration using System Variable (-D)